ote/src/cljs/ote/style/base.cljs

```diff
@@ -240,6 +240,26 @@
    :padding-right "0"
    :overflow-wrap "break-word"})
 
+(def new-change-indicator
```

Comment:
`style/transit_changes.cls` is the shared style for `transit-changes` and `transit-visualization`. Should these be moved there?